msys2-runtime-3.3/PKGBUILD

```diff
@@ -278,9 +278,10 @@ build() {
 
   (cd "${srcdir}/msys2-runtime/winsup" && ./autogen.sh)
 
+  local CYGWIN_CHOST="${CHOST/-msys/-cygwin}"
   "${srcdir}"/msys2-runtime/configure \
     --prefix=/usr \
-    --build=${CHOST} \
+    --build=${CYGWIN_CHOST} \
```

Comment:
I am not sure what this build failure is about:

> ```
> 2025-06-16T09:01:58.5322130Z ==> Starting package_msys2-runtime-3.3-devel()...
> 2025-06-16T09:01:58.5946461Z cp: cannot stat '/d/_/msys2-runtime-3.3/src/dest/usr/x86_64-pc-msys/include': No such file or directory
> ```

But maybe it required an `s/CHOST/CYGWIN_CHOST/` also here: https://github.com/msys2/MSYS2-packages/blob/8d47a795135ad3f5ac0fe85253485ae76d1ade4f/msys2-runtime-3.3/PKGBUILD#L333

Reply:
And of course here, too: https://github.com/msys2/MSYS2-packages/blob/8d47a795135ad3f5ac0fe85253485ae76d1ade4f/msys2-runtime-3.3/PKGBUILD#L333

Reply:
Ah, yes, thanks. I only tested with 3.6 locally

Reply:
> Ah, yes, thanks. I only tested with 3.6 locally

and restarted the env between builds, which is currently missing in our CI script.